Migration.md

```diff
@@ -5,6 +5,10 @@ Deprecated code produces compile-time warnings. These warning serve as
 notification to users that their code should be upgraded. The next major
 release will remove the deprecated code.
 
+## Ignition Gazebo 5.x to 6.x
+
+*
```

Comment:
?

Reply:
Left there as a reminder for us to fill it once we settle on the API